pipelines/asana_dlt/const.py

```diff
@@ -0,0 +1,142 @@
+# Default start date for Asana API requests, only tasks started after this date will be collected
+DEFAULT_START_DATE = "2010-01-01T00:00:00.000Z"
+
+# Asana API request timeout
+TIMEOUT = 300
```

Comment:
should be passed as an argument maybe?

Reply:
hm, I would have this as a setting across all endpoints and keep it like this for now